django/core/jinja2/core/events/retrieve.jinja

```diff
@@ -75,6 +75,7 @@
         <a href="{{ url('core:event-edit', pk=id) }}">
             <div class="btn btn-primary my-1 w-100">Edit</div>
         </a>
+        {{ mark_spam_confirm_modal("Mark spam", title, url("core:event-mark-spam", pk=id), csrf_input) }}
```

Comment:
should update the conditional rendering of the mark spam buttons to only show when the content object `is_marked_spam` is `False` and that the viewing user has permission